packages/itp-rcc-button/src/Button.js

```diff
@@ -0,0 +1,178 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+function insertSpace(child) {
+  if (child == null) {
+    return
+  }
+
+  if (typeof child === 'string') {
+    return <span>{child}</span> // eslint-disable-line consistent-return
+  }
+
+  return child // eslint-disable-line consistent-return
+}
+
+export default class Button extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: props.loading,
+    }
+
+    this.handleClick.bind(this)
```

Comment:
I rather prefer not to bind this, but to use an arrow function for the handleClick method

Reply:
In that case you can get rid of the constructor and use ```state = { loading: props.loading }```

Reply:
Agreed, same comment as below. We'll need to check changes needed in `itp-react-scripts`